translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.translate;
+
+// [START translate_hybrid_imports]
+import com.google.cloud.texttospeech.v1.AudioConfig;
+import com.google.cloud.texttospeech.v1.AudioEncoding;
+import com.google.cloud.texttospeech.v1.SsmlVoiceGender;
+import com.google.cloud.texttospeech.v1.SynthesisInput;
+import com.google.cloud.texttospeech.v1.SynthesizeSpeechResponse;
+import com.google.cloud.texttospeech.v1.TextToSpeechClient;
+import com.google.cloud.texttospeech.v1.VoiceSelectionParams;
+
+import com.google.cloud.translate.v3beta1.CreateGlossaryRequest;
+import com.google.cloud.translate.v3beta1.GcsSource;
+import com.google.cloud.translate.v3beta1.Glossary;
+import com.google.cloud.translate.v3beta1.Glossary.LanguageCodesSet;
+import com.google.cloud.translate.v3beta1.GlossaryInputConfig;
+import com.google.cloud.translate.v3beta1.GlossaryName;
+import com.google.cloud.translate.v3beta1.LocationName;
+import com.google.cloud.translate.v3beta1.TranslateTextGlossaryConfig;
+import com.google.cloud.translate.v3beta1.TranslateTextRequest;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Feature.Type;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+
+import com.google.common.html.HtmlEscapers;
+import com.google.protobuf.ByteString;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+// [END translate_hybrid_imports]
+
+
+public class HybridGlossaries {
+
+  // [START translate_hybrid_project_id]
+  private static final String projectId = System.getenv("PROJECT_ID");
+  // [END translate_hybrid_project_id]
+
+  // [START translate_hybrid_vision]
+  /**
+   * @param filePath input image file
+   **/
+  static String picToText(String filePath) {
+    List<AnnotateImageRequest> requests = new ArrayList<>();
+
+    try (ImageAnnotatorClient client = ImageAnnotatorClient.create()) {
```

Comment:
Can you add this comment to [client initialization](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#client-initialization): 
```
  // Initialize client that will be used to send requests. This client only needs to be created
  // once, and can be reused for multiple requests. After completing all of your requests, call
  // the "close" method on the client to safely clean up any remaining background resources.
```